cmd/courses-assignments-list/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/courses/assignments"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	CourseID string `help:"ID of the course." arg:""`
+	Table    bool   `help:"Output data as a TSV." default:"false"`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("List assignments from a specific course."),
```

Comment:
I suggest the description should be: "List the assignments from the course."

Since they specify the course, they should understand it is for a specific course.